packages/node/src/handlers.ts

```diff
@@ -272,11 +272,23 @@ function getStatusCodeFromResponse(error: MiddlewareError): number {
   return statusCode ? parseInt(statusCode as string, 10) : 500;
 }
 
+/** JSDoc */
```

Comment:
```suggestion
/** Returns true if response code is internal server error */
```

To nitpick anything.